scenarios/hci.yaml

```diff
@@ -0,0 +1,64 @@
+---
+undercloud:
+  config:
+    - section: DEFAULT
+      option: undercloud_hostname
+      value: undercloud.localdomain
+    - section: DEFAULT
+      option: undercloud_timezone
+      value: UTC
+    - section: DEFAULT
+      option: undercloud_debug
+      value: true
+    - section: DEFAULT
+      option: container_cli
+      value: podman
+    - section: DEFAULT
+      option: undercloud_enable_selinux
+      value: false
+    - section: DEFAULT
+      option: cleanup
+      value: false
+  undercloud_parameters_override: "hci/hieradata_overrides_undercloud.yaml"
+  undercloud_parameters_defaults: "hci/undercloud-parameter-defaults.yaml"
```

Comment:
since we're starting from scratch here maybe we can use either - or _ in the env files. like pick one and be consistent.

Reply:
thanks @marios I've changed the files to all use underscores